File: remote/src/main/java/com/kelsos/mbrc/annotations/Repeat.java
package com.kelsos.mbrc.annotations;

import android.support.annotation.StringDef;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

public class Repeat {
  public static final String ALL = "all";
  public static final String NONE = "none";
  public static final String ONE = "one";
  public static final String CHANGE = "";
  public static final String UNDEFINED = "undef";

  private Repeat() {
    //no instance
  }

  @StringDef({
      ALL,
      NONE,
      ONE,
      CHANGE,
      UNDEFINED
  })
  @Retention(RetentionPolicy.SOURCE) public @interface Mode {
  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/content/active_status/Repeat.kt
package com.kelsos.mbrc.content.active_status


object Repeat {

  const val ALL = "all"
  const val NONE = "none"
  const val ONE = "one"

  @android.support.annotation.StringDef(com.kelsos.mbrc.content.active_status.Repeat.ALL, com.kelsos.mbrc.content.active_status.Repeat.NONE, com.kelsos.mbrc.content.active_status.Repeat.ONE)
  @Retention(AnnotationRetention.SOURCE)
  annotation class Mode

}
